usr/share/rear/verify/BAREOS/default/040_prepare_environment_vars.sh

```diff
@@ -0,0 +1,24 @@
+# read config vars defined during prep.
+source "$VAR_DIR/bareos.conf"
```

Comment:
Like I said above, I would like to challenge the need for this file and propose to have all of those settings be part of the regular ReaR configuration instead.

Reply:
please use existing `rescue.conf` for the stuff that is needed